test/functions.jl

```diff
@@ -204,4 +205,149 @@
             @test g.affine_terms == MOI.VectorAffineTerm.([1, 2, 1], MOI.ScalarAffineTerm.([3, 1, 1], [x, x, y]))
         end
     end
+    @testset "Conversion to canonical form" begin
+        function _isapprox(f1::T, f2::T) where {T <: Union{MOI.ScalarAffineFunction, MOI.VectorAffineFunction}}
```

Comment:
The difference between `_isapprox` and `isapprox` ([here](https://github.com/JuliaOpt/MathOptInterface.jl/blob/ae87890bec8922c21df4c817b4e847c67b402240/src/functions.jl#L310)) is confusing and not well explained by an underscore. Maybe call this `isapprox_ordered`?

Reply:
yup, done